rocAL/rocAL/include/decoders/image/fused_crop_decoder.h

```diff
@@ -83,4 +84,14 @@ class FusedCropTJDecoder : public Decoder {
     };
     bool _is_partial_decoder = true;
     std::vector <float> _bbox_coord;
+    std::mt19937 _rand_gen;
+    int64_t getseed() { return ParameterFactory::instance()->get_seed(); }
+    void generate_rngs(int64_t seed, int64_t N) {
+      std::seed_seq seq{seed};
+      std::vector<int64_t> seeds(N);
+      seq.generate(seeds.begin(), seeds.end());
+      for (int64_t i = 0; i < N; i++) {
```

Comment:
Not sure what this code is doing. Looks like it is setting seeds for the same random gen